gc3libs/__init__.py

```diff
@@ -1818,19 +1818,21 @@ def info():
           u'a second message ...'
 
         """
+        return locals()
 
-        def fget(self):
-            return self.history.last()
-
-        def fset(self, value):
+    @info.setter
+    def info(self, value):
+        try:
+            self.history.append(str(value, errors='replace'))
+        except (TypeError, ValueError):
             try:
-                self.history.append(str(value, errors='replace'))
-            except (TypeError, ValueError):
-                try:
-                    self.history.append(str(value))
-                except Exception as err:
-                    log.error("Cannot append `%s` to history of task %s", value, self)
-        return locals()
+                self.history.append(str(value))
+            except Exception as err:
+                log.error("Cannot append `%s` to history of task %s", value, self)
+
+    @info.getter
```

Comment:
`info.getter` is not a decorator; see comment at line 1803 and 1821 for how to translate `defproperty` into `property`.

Reply:
Will fix asap.